manager/subscription/heartbeat_manager.py

```diff
@@ -80,7 +84,7 @@ async def dispatch_heartbeats(cls):
             channel_layer = get_channel_layer()
             while True:
                 try:
-                    print("sending data")
+                    print("sending data", flush=True)
```

Comment:
This print is for log reviews?

Reply:
Yes, to check when a heartbeat message is sent

Reply:
Yes, to check when a heartbeat message is sent